lib/chef/knife/solo_cook.rb

```diff
@@ -260,14 +260,15 @@ def write(content, dest)
         file.unlink
       end
 
-      def rsync(source_path, target_path, extra_opts = '--delete-after')
+      def rsync(source_path, target_path, extra_opts = ['--delete-after', '-zt'])
         if config[:ssh_gateway]
           ssh_command = "ssh -TA #{config[:ssh_gateway]} ssh -T -o StrictHostKeyChecking=no #{ssh_args}"
         else
           ssh_command = "ssh #{ssh_args}"
         end
 
-        cmd = ['rsync', '-rL', rsync_debug, rsync_permissions, %Q{--rsh=#{ssh_command}}, extra_opts]
+        cmd = ['rsync', '-rL', rsync_debug, rsync_permissions, %Q{--rsh=#{ssh_command}}]
+        cmd += extra_opts
```

Comment:
Why '+=' instead of using the insertion operator? I would tend to be consistent and use one or the other.